cryptoconditions/types/ed25519.py

```diff
@@ -154,4 +154,4 @@ def validate(self, message=None, **kwargs):
         if not (message and self.signature):
             return False
 
-        return self.public_key.verify(message, self.signature, encoding=None)
+        return self.public_key.verify(message, self.signature, encoding='bytes')
```

Comment:
Should probably update the docstring to specific that `message` has to be a bytestring.